pilz_robot_tfc_api/src/tfc_api.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""API for easy usage of Pilz robot test facility."""
+
+OP_MODE_T1 = 0
+OP_MODE_T2 = 1
+OP_MODE_AUTO = 2
```

Comment:
Please add documentation or name the constants in a way that makes clear, what they should be used for.

Reply:
Changed to enum => Done.